crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -100,9 +100,8 @@ impl EventTimelineItem {
     ) -> Option<EventTimelineItem> {
         use super::traits::RoomDataProvider;
 
-        let sync_event = latest_event.into_event();
```

Comment:
Can `LatestEvent::into_event` be removed then?

Reply:
Resolved with next commits.